app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -6,6 +6,7 @@ import type { PII } from '../services/upload';
 import AnalyticsContext from '../context/analytics';
 import BarcodeAttentionWarning from './barcode-attention-warning';
 import FailedCaptureAttemptsContext from '../context/failed-capture-attempts';
+import FeatureFlagContext from '../context/feature-flag';
```

Comment:
same as above re: which context is needed

Reply:
✅ https://github.com/18F/identity-idp/pull/10156#discussion_r1517894124